lib/includes.js

```diff
@@ -0,0 +1,189 @@
+var Promise = require('bluebird'),
```

Comment:
This code isn't ES6. Use ```const``` if the var is immutable and ```let``` when it has to be. ```var``` when you really don't want to use block scoping (almost never).